platform-service/src/unstract/platform_service/extensions.py

```diff
@@ -0,0 +1,3 @@
+from peewee import PostgresqlDatabase
+
+db = PostgresqlDatabase(None)
```

Comment:
@Deepak-Kesavan earlier in the code we were passing some Envs during initial construction itself. Does changing it like this could cause any issues?

Reply:
It is not an issue @ritwik-g . We are doing the same in prompt service.

https://github.com/Zipstack/unstract/blob/4ee004e9ba64c6ce44e060d8dab765d693a551ab/prompt-service/src/unstract/prompt_service/config.py#L31